src/test/java/com/exasol/bucketfs/SyncAwareBucketIT.java

```diff
@@ -168,4 +167,13 @@ void testReplaceFile(@TempDir final Path tempDir) throws InterruptedException, B
             }
         }
     }
+
+    @Test
+        //[itest->dsn~delete-a-file-from-a-bucket~1]
+    void testDeleteFile() throws BucketAccessException, InterruptedException, TimeoutException {
+        final var bucket = getDefaultBucket();
+        bucket.uploadStringContent("test", "myFile.txt");
+        bucket.deleteFileNonBlocking("myFile.txt");
+        assertThat(bucket.listContents(), not(hasItem("myFile.txt")));
```

Comment:
Interesting. This means that the content manifest is updated before the actual deletion.
I think, we will soon need a blocking version of this. But let's do that in a separate ticket.